src/main.rs

```diff
@@ -8,12 +8,12 @@
  * except according to those terms.
  */
 
-extern crate cargo_ebuild;
-extern crate cargo;
+#[macro_use]
+extern crate human_panic;
```

Comment:
So if someone inputted the wrong arguments we're not going to show them the following:

```
Well, this is embarrassing.

cargo-ebuild had a problem and crashed. To help us diagnose the problem you can send us a crash report.

We have generated a report file at "/var/folders/zw/bpfvmq390lv2c6gn_6byyv0w0000gn/T/report-8351cad6-d2b5-4fe8-accd-1fcbf4538792.toml". Submit an issue or email with the subject of "human-panic Crash Report" and include the report as an attachment.

- Homepage: https://github.com/cardoe/cargo-ebuild
- Authors: Doug Goldstein <cardoe@cardoe.com>

We take privacy seriously, and do not perform any automated error collection. In order to improve the software, we rely on people to submit reports.

Thank you kindly!
```

This is the wrong crate for the job. Printing out a reasonable error message worked previously and this isn't reasonable.